code/game/objects/structures/crates_lockers/crates/secure.dm

```diff
@@ -19,8 +19,9 @@
 	if(user)
 		to_chat(user, span_danger("The crate's anti-tamper system activates!"))
 		log_bomber(user, "has detonated a", src)
-	for(var/atom/movable/AM in src)
-		qdel(AM)
+	for(var/atom/movable/content as anything in src)
+		if(!isliving(content))
```

Comment:
This loop just shouldn't exist at all. I could get around this with a body bag.

Reply:
Or you should be calling the procs to deal damage to objects, rather than letting qdel be called.